Makefile

```diff
@@ -58,25 +58,24 @@ update:
 /tmp/antlr3_python3_runtime_3.4:
```

Comment:
This path is temporary and cleaned up on reboot. Why don't you check for a file where the antlr runtime has actually been installed ?

Reply:
Hmmm, turns out it only install python libraries, no binary, so the location would depend on the virtualenv setup and the python version used... I might take the option to download and extract in the working directory to avoid multiple downloads... until all pip requirements are managed directly via python/pip.